pkg/controller/workload/workload_cache.go

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2024 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package workload
+
+import (
+	"encoding/binary"
+	"sync"
+
+	workloadapi "kmesh.net/kmesh/api/v2/workloadapi"
+)
+
+var (
+	workloadCache = workloadStore{
```

Comment:
Nit: the code style of using global var is not recommended in go.

Reply:
fixed